modules/ROOT/pages/database-administration/syntax.adoc

```diff
@@ -0,0 +1,251 @@
+----
+CREATE CONSTRAINT [constraint_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+REQUIRE n.propertyName IS NOT NULL
+----
+
+[[administration-syntax-database-management]]
+== Database management command syntax
+
+The below table covers both standard and composite databases. 
+
+[options="header", width="100%", cols="1m,5a"]
+|===
+| Command | Syntax
+
+| SHOW DATABASE
+|
+[source, syntax, role="noheader"]
+----
+SHOW { DATABASE[S] name \| DATABASE[S] \| DEFAULT DATABASE \| HOME DATABASE }
+[WHERE expression]
+----
+
+[source, syntax, role="noheader"]
+----
+SHOW { DATABASE[S] name \| DATABASE[S] \| DEFAULT DATABASE \| HOME DATABASE }
+YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]
+[WHERE expression]
+[RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+----
+
+| CREATE DATABASE
+|
+[source, syntax, role="noheader"]
+----
+CREATE DATABASE name [IF NOT EXISTS]
+[TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]]
+[OPTIONS "{" option: value[, ...] "}"]
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+[source, syntax, role="noheader"]
+----
+CREATE OR REPLACE DATABASE name
+[TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]]
+[OPTIONS "{" option: value[, ...] "}"]
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+| CREATE COMPOSITE DATABASE
+|
+[source, synatx, role="noheader"]
+----
+CREATE COMPOSITE DATABASE name [IF NOT EXISTS]
+[OPTIONS "{" "}"]
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+[source, syntax, role="noheader"]
+----
+CREATE OR REPLACE COMPOSITE DATABASE name
+[OPTIONS "{" "}"]
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+| ALTER DATABASE
+|
+[source, syntax, role="noheader"]
+----
+ALTER DATABASE name [IF EXISTS]
+{
+SET ACCESS {READ ONLY \| READ WRITE} \|
+SET TOPOLOGY n PRIMAR{Y\|IES} [m SECONDAR{Y\|IES}]
+}
+[WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+| STOP DATABASE
+|
+[source, syntax, role="noheader"]
+----
+STOP DATABASE name [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+| START DATABASE
+|
+[source, syntax, role="noheader"]
+----
+START DATABASE name [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+| DROP DATABASE
+|
+[source, syntax, role="noheader"]
+----
+DROP [COMPOSITE] DATABASE name [IF EXISTS] [{DUMP\|DESTROY} [DATA]] [WAIT [n [SEC[OND[S]]]]\|NOWAIT]
+----
+
+|===
+
+[[administration-syntax-database-alias-management]]
+== Database alias management command syntax
+
+[options="header", width="100%", cols="1,5a"]
+|===
+| Command | Syntax
+| Show Database Alias
+|
+[source, syntax, role=noheader]
+-----
+SHOW ALIAS[ES] [name] FOR DATABASE[S]
+[WHERE expression]
+-----
+[source, syntax, role=noheader]
+-----
+SHOW ALIAS[ES] [name] FOR DATABASE[S]
+YIELD { * \| field[, ...] } [ORDER BY field[, ...]] [SKIP n] [LIMIT n]
+[WHERE expression]
+[RETURN field[, ...] [ORDER BY field[, ...]] [SKIP n] [LIMIT n]]
+-----
+Lists both local and remote database aliases, optionally filtered on the alias name.
+
+| Create Local Alias
+|
+[source, syntax, role=noheader]
+-----
+CREATE ALIAS name [IF NOT EXISTS] FOR DATABASE targetName
```

Comment:
while the syntax table at the top of the cypher manual alias page only have `name`, the more specific syntax examples in the create and alter sections have `[compositeDatabaseName.]aliasName`. I think it would make sense to have `[compositeDatabaseName.]aliasName` for all alias names in the syntax section here (not just alter and create)